src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/ManageApprenticesOrchestrator.cs

```diff
@@ -26,11 +23,10 @@
 using SFA.DAS.ProviderApprenticeshipsService.Application.Exceptions;
 using SFA.DAS.ProviderApprenticeshipsService.Application.Queries.GetCommitment;
 using SFA.DAS.ProviderApprenticeshipsService.Web.Validation;
-using SFA.DAS.ProviderApprenticeshipsService.Domain.Models.ApprenticeshipCourse;
 
 namespace SFA.DAS.ProviderApprenticeshipsService.Web.Orchestrators
 {
-    public sealed class ManageApprenticesOrchestrator
+    public sealed class ManageApprenticesOrchestrator : BaseCommitmentOrchestrator
     {
         private readonly IMediator _mediator;
```

Comment:
Same here, we can use Mediator, HashingService and Logger from the base class instead.